atmosphere/atmospherex/atmospherex_api.py

```diff
@@ -0,0 +1,75 @@
+import math
+from datetime import datetime
+from typing import Generator
+
+import requests
+
+LIMIT = 100
```

Comment:
```suggestion
PREDICTION_PAGE_SIZE = 100
```

I'd suggest something more specific, I don't know the perfect name for it but maybe something like this?

`LIMIT` is too generic